src/button/Button.d.ts

```diff
@@ -21,9 +22,17 @@ export type ButtonProps = {
     fluid?: boolean,
     circular?: boolean,
     textOverflow?: React.ComponentClass<object>, // todo(typo) when translate text overflow, update here its props types
-    style?: React.CSSProperties,
+    style?: CSSProperties,
     className?: string
   }
 
```

Comment:
also, there where tons of validations (`propsInvariants`), probably impossible to replicate in TS, but I'd add a TODO as usual